src/_components/Navbar.jsx

```diff
@@ -1,29 +1,109 @@
 import React from "https://esm.sh/react";
 
-export default function Navbar() {
+// Global navigation links array
+const navigationLinks = [
+  { href: "/mission", text: "Mission" },
+  { href: "/projects", text: "Projects" },
+  { href: "/community", text: "Get Involved" },
+  { href: "/blog", text: "Blog" },
+];
+
+export default function NavigationBar() {
   const logo = "/assets/logos/logo_negative.png";
   return (
-    <nav className="flex items-center justify-around space-x-64 p-6 bg-primary z-50">
-      <a href="/" className="flex items-center text-white mr-60">
-        <img src={logo} alt="Logo" className="mr-2 w-10 h-10" />
-        <span className="self-center text-3xl font-semibold font-poppins whitespace-nowrap dark:text-white">
+    <nav className="relative flex items-center justify-between p-6 bg-primary z-50">
+      {/* Logo and Brand Name Section */}
+      <a href="/" className="flex items-center text-white">
+        <img src={logo} alt="Blueprint Logo" className="mr-2 w-10 h-10" />
+        <span className="text-3xl font-poppins font-semibold hover:font-bold">
           blueprint
         </span>
```

Comment:
Can you change the navbar and buttons to be larger on mobile? Mobile users are using their thumb instead of a cursor so usability is something to keep in mind 😃 

Something like this
```
<nav className="relative flex items-center justify-between lg:p-6 p-10 bg-primary z-50">
  {/* Logo and Brand Name Section */}
  <a href="/" className="flex items-center text-white">
  <img src={logo} alt="Blueprint Logo" className="mr-2 lg:w-10 lg:h-10 w-16 h-16" />
  <span className="lg:text-3xl text-5xl font-poppins font-semibold hover:font-bold">
    blueprint
  </span>
  </a>
    ...
```

**Note**: you'd also have to increase the size of the text in the dropdown and the hamburger menu itself (`w-6` to `w-10`) too